pkg/clients/aws.go

```diff
@@ -643,3 +643,18 @@ func CleanError(err error) error {
 func Wrap(err error, msg string) error {
 	return errors.Wrap(CleanError(err), msg)
 }
+
```

Comment:
I first thought that it would make the code more readable, but after implementing your review feedback, I don't think this is true anymore.

I removed those functions.